pages/web-literacy/Skills.jsx

```diff
@@ -6,14 +6,18 @@ var makeLinksFromWebLitSkills = require("./MakeLinksFromWebLitSkills.jsx");
 var WhitePaperLink = require("./WhitePaperLink.jsx");
 
 var Skills21CPage = React.createClass({
+  contextTypes: {
+    intl: React.PropTypes.object
+  },
   statics: {
     pageTitle: '21st Century Skills',
     pageClassName: 'web-literacy-skills-page'
   },
   generateSkillItem: function(skill) {
+    var that = this;
```

Comment:
this should not be necessary, whatever ends up using `that` should be a component function, with the `this` binding left to React.